src/olympia/devhub/templates/devhub/emails/verify-email-requested.ltxt

```diff
@@ -3,7 +3,7 @@
 You're receiving this message because you requested to verify your email.
 To finish this process, please click on the link below:
 
-{{ confirmation_link }}
+{{ settings.SITE_URL }}{{ confirmation_link }}
```

Comment:
We have a `absolutify` filter that takes care of this, it handles the URL already starting with `http://` or `https://`* and works even when `SITE_URL` has been overridden (shouldn't happen in your specific case, it's for addons-server containers meant for admin tools behind VPN, but still)

It can be called as a function when generating context - which might be better in this case, cause emails are processed with Django Template Langage, and I'm not sure whether or not we register that filter outside of Jinja2

Reply:
Done.